WORKSPACE

```diff
@@ -185,13 +187,12 @@ nixpkgs_package(
 # Use the go distribution from nixpkgs instead of letting rules_go
 # download some arbitrary binary distribution. This fixes buildifier
 # on both NixOS and Darwin.
-go_wrap_sdk(
+# If the environment variable DONT_WRAP_GO_SDK is set, it will
```

Comment:
I rather prefer a positive variable name, such as `WRAP_GO_SDK` which default to one, instead of the inverted logic.